.gitignore

```diff
@@ -42,3 +42,4 @@ coverage.xml
 docs/_build/
 
 /ckanext/switzerland/i18n/*/LC_MESSAGES/ckanext-switzerland.mo
+.idea
```

Comment:
this should go in your global gitignore file - see https://gist.github.com/subfuzion/db7f57fff2fb6998a16c